src/conmon.c

```diff
@@ -173,6 +175,12 @@ int main(int argc, char *argv[])
 			g_unix_fd_add(winsz_fd_r, G_IO_IN, ctrl_winsz_cb, NULL);
 	}
 
+	if (opt_seccomp_notify_socket != NULL) {
+		if (opt_seccomp_notify_plugins == NULL)
```

Comment:
we should probably also check if we have USE_SECCOMP here

Reply:
and fail if so?

Reply:
good idea, fixed now